apitest/src/test/java/bisq/apitest/method/payment/AbstractPaymentAccountTest.java

```diff
@@ -0,0 +1,141 @@
+package bisq.apitest.method.payment;
+
+import bisq.core.api.model.PaymentAccountForm;
+import bisq.core.locale.Res;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.stream.JsonWriter;
+
+import java.nio.file.Paths;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+
+import static bisq.apitest.config.BisqAppConfig.alicedaemon;
+import static java.lang.String.format;
+import static java.lang.System.getProperty;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+
+
+import bisq.apitest.method.MethodTest;
+
+@Slf4j
+public class AbstractPaymentAccountTest extends MethodTest {
+
+    static final String PROPERTY_NAME_COMMENT = "_COMMENT_";
+    static final String PROPERTY_VALUE_COMMENT = "Please do not edit the paymentMethodId field.";
```

Comment:
I don't think there is a need to be polite. Better with something like `"Do not manually edit the paymentMethodId field"`

Reply:
OK.  Politeness will be removed.

Reply:
Fixed in commit 62a2aa85434213493c6d5225de9b694123ad67c4.